handler/src/main/java/io/netty5/handler/ssl/PemReader.java

```diff
@@ -95,7 +94,7 @@
             throw new CertificateException("found no certificates in input stream");
         }
 
-        return certs.toArray(new ByteBuf[0]);
+        return certs.toArray(ByteBuf[]::new);
```

Comment:
TIL about this syntax :)

Reply:
It's a method reference to the array constructor. 😉